lib/cli/commands/restore.ts

```diff
@@ -0,0 +1,86 @@
+import readline from 'readline';
+import { Arguments } from 'yargs';
+import { RestoreNodeRequest, RestoreNodeResponse } from '../../proto/xudrpc_pb';
+import { callback, loadXudInitClient } from '../command';
+import { getDefaultCertPath, waitForCert } from '../utils';
+
+export const command = 'restore';
+
+export const describe = 'restore an xud instance from seed';
+
+export const builder = {};
+
+const formatOutput = (response: RestoreNodeResponse.AsObject) => {
+  let walletRestoredMessage = 'The following wallets were restored: ';
+
+  if (response.restoredLndsList.length) {
+    walletRestoredMessage += response.restoredLndsList.join(', ');
+  }
+
+  if (response.restoredRaiden) {
+    if (!walletRestoredMessage.endsWith(' ')) {
+      walletRestoredMessage += ', ';
+    }
+
+    walletRestoredMessage += 'ERC20(ETH)';
+  }
+
+  console.log(walletRestoredMessage);
+};
+
+export const handler = (argv: Arguments<any>) => {
+  const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+    terminal: true,
+  });
+
+  console.log(`
+You are restoring an xud node key and underlying wallets. All will be secured by
+a single password provided below.
+  `);
+  rl.question('Enter your 24 word mnemonic: ', (mnemonicStr) => {
```

Comment:
Should we clarify here that the words need to be separated by space?

Reply:
Not a bad idea, I'll do that.